pkg/kubelet/images/image_gc_manager.go

```diff
@@ -333,6 +333,14 @@ func (im *realImageGCManager) GarbageCollect(ctx context.Context, beganGC time.T
 	if fsStats.CapacityBytes != nil {
 		capacity = int64(*fsStats.CapacityBytes)
 	}
+
+	// Check valid capacity.
```

Comment:
this sequencing losing an important piece of information - what is the value of `available`. In the old sequencing we knew that `available==0`. In a new one it may either be valid or also be `0`. So I prefer the old way.

/close

please reopen if you want to change sequencing, but in this case - include the available memory in the error message saying that the capacity is incorrect.